content/02.introduction.md

```diff
@@ -155,14 +155,41 @@ Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly di
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
-Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.
-[@doi:10.1001/jamapediatrs.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.
-Most of the children reviewed had mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose or were asymptomatic.
-Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
-Radiologic findings were not always mentioned, but when mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions.
-The results of COVID-19 infection can vary greatly among pediatric patients as well as adults.
-In particular, some children have experienced a severe inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069;@doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117].
-Heart failure can also occur [@doi:10.1161/CIRCULATIONAHA.120.048360].
+#### Subpopulations of Special Concern  
+
+In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, of older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].
```

Comment:
```suggestion
A number of interconnected factors appear to influence population-level trends in the impact and severity of SARS-CoV-2.
In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, of older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].
```

Reply:
Hello @rando2 !  I was thinking about the term "minoritized."  I have heard it before, but it sounds very awkward as a neologism to me -- I assume it is supposed to mean "marginalized," but I'm not sure why people don't use that word instead if that's what they mean.  I thought of "minority" as being a statistical word, just meaning a group is smaller in number than some other group.  But in that case, I don't know of any data for minoritized groups in the United States OTHER than African American, Hispanic or Native American.  Whether, say, Muslim Jewish or LGBT or autistic people are at higher risk for COVID because some people mistreat them, is an empirical question which as far as I know nobody asked.  It might be true but we don't have any data.  So I think it might make more sense to just say "Hispanic/Latino and Native American" and just drop "other minoritized populations such as."  Would that be okay?

Reply:
In other words, since we only know about those three groups, maybe they should be the only ones we mention.

Reply:
I did just recently read about a similar but far larger study done in the UK:
https://doi.org/10.1038/s41586-020-2521-4
They were actually able to estimate P(death) instead of P(death | diagnosis) because they used a massive database of millions of medical records.  The findings were similar but sturdier than the papers we cite here, because of the larger and less selected sample.  So I will add a sentence about it if it is okay with you.  Sorry to keep adding references!  I will add it soon so that you can do the merging tonight if you want.